tools/tokenserver/purge_old_records.py

```diff
@@ -125,15 +158,18 @@ def delete_service_data(user, secret, timeout=60, dryrun=False):
     remove any data it still has stored for the user.  We simulate a DELETE
     request from the user's own account.
     """
-    token = tokenlib.make_token({
-        "uid": user.uid,
-        "node": user.node,
-        "fxa_uid": user.email.split("@", 1)[0],
-        "fxa_kid": format_key_id(
-            user.keys_changed_at or user.generation,
-            binascii.unhexlify(user.client_state)
-        ),
-    }, secret=secret)
+    token = tokenlib.make_token(
+        {
+            "uid": user.uid,
+            "node": user.node,
+            "fxa_uid": user.email.split("@", 1)[0],
+            "fxa_kid": format_key_id(
+                user.keys_changed_at or user.generation,
+                binascii.unhexlify(user.client_state),
+            ),
+        },
+        secret=secret,
+    )
     secret = tokenlib.get_derived_secret(token, secret=secret)
     endpoint = PATTERN.format(uid=user.uid, node=user.node)
     auth = HawkAuth(token, secret)
```

Comment:
Mostly just to exercise code. For  a dry run, I prefer doing as much as possible up until I'm about to do something destructive / permanent so that I know that all other aspects/functions work. Ideally, the `dryrun` would dump state so that could see what the action to be taken would be, but that can produce a lot of noise.

Reply:
Ah that's a great point!